config/crd/bases/carto.run_clustertemplates.yaml

```diff
@@ -199,6 +199,17 @@ spec:
                       unhealthy. Otherwise, healthiness is Unknown.
                     type: string
                 type: object
+              lifecycle:
+                default: default
+                description: Lifecycle specifies whether template modifications should
```

Comment:
Nit: I know it's still dependent on @zrob, but I would like this to be clearly marked `Experimental`